docs/source/users/index.md

```diff
@@ -349,14 +349,20 @@ You may now select a chosen Bedrock model from the drop-down menu box title `Com
 
 If your provider requires an API key, please enter it in the box that will show for that provider. Make sure to click on `Save Changes` to ensure that the inputs have been saved.
 
-Bedrock also allows custom models to be trained from scratch or fine-tuned from a base model. Jupyter AI enables a custom model to be called in the chat panel using its `arn` (Amazon Resource Name). The interface is shown below:
+Bedrock also allows custom models to be trained from scratch or fine-tuned from a base model. Jupyter AI enables a custom model to be called in the chat panel using its `arn` (Amazon Resource Name, ARN). The interface is shown below:
 
 <img src="../_static/bedrock-chat-custom-model-arn.png"
     width="75%"
     alt='Screenshot of the Jupyter AI chat panel where the custom model is selected using model arn.'
     class="screenshot" />
 
-For detailed workflows, see [Using Amazon Bedrock with Jupter AI](bedrock.md).
+Note that:
+1. Bedrock Base models: All available models will already be available in the drop down model list. The above interface also allows use of base model IDs or ARNs, though this is unnecessary as they are in the dropdown list.
+2. Bedrock Custom models: If you have fine-tuned a Bedrock base model you may use the ARN for this custom model. Make sure to enter the correct provider information, such as `amazon`, `anthropic`, `cohere`, `meta`, `mistral` (always in lower case).
+3. Provisioned Models: These are models that run on dedicated endpoints. Users can purchase Provisioned Throughput Model Units to get faster throughput. These may be base or custom models. Enter the ARN for these models in the Model ID field.
+4. Cross-region Inference: Amazon Bedrock now permits cross-region inference, where a model hosted in a different region than that of the user may be specified, see the [inference profiles documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/inference-profiles-support.html). Such models have IDs prefixed by a region identifier such as `us.meta.llama3-2-1b-instruct-v1:0`, for example. To use this feature, simply enter the Inference profile ID for the cross-region model instead of the ARN.
```

Comment:
I understand that having a summary of the Bedrock page on the main `users/index.md` page may be helpful. My concern is that this may do more harm than good, because it buries the link to the Bedrock docs page within several paragraphs & screenshots. It's not easy for users to find the link that takes a user to the Bedrock docs page.

Can we replace this entire section with a single sentence so that there's a single source of truth regarding how to use Jupyter AI with Amazon Bedrock?

```
See our dedicated documentation page on using Jupyter AI with Amazon Bedrock: ...
```